src/main/java/com/google/sps/servlets/GetSessionServlet.java

```diff
@@ -12,6 +23,26 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    throw new Error("Unimplemented");
+    DatastoreClientInterface datastoreClient = new DatastoreClient();
+    String sessionId = 
+        URLDecoder.decode(request.getParameter("session-id"), StandardCharsets.UTF_8);
+    String name = URLDecoder.decode(request.getParameter("name"), StandardCharsets.UTF_8);
+    AttendeeInterface updatedAttendee = new Attendee(sessionId, name, new Date());
+    datastoreClient.insertOrUpdateAttendee(updatedAttendee);
```

Comment:
i dont understand this. Why do we need this?

Reply:
hmm I thought the date was supposed to be updated for the background task manager

Reply:
right. Please add a comment indicating that that's what this bit of code is doing.